lib/Command/SharesFiles.php

```diff
@@ -203,15 +203,20 @@ private function displayShares(
 					$sharedTo .= '@' . $recipient->getInstance();
 				}
 			}
-			$circle = $share->getCircle();
+			$circleField = '';
+			if ($share->hasCircle()) {
```

Comment:
I don't quite understand when a `ShareWrapper` has a circle and when it does not. We might also want to skip all `$share`s without a circle.